code/modules/client/preference/preferences_mysql.dm

```diff
@@ -18,7 +18,7 @@
 					ghost_anonsay,
 					exp,
 					clientfps,
-					atklog
+					atklog,
```

Comment:
hmmm

Reply:
Will fix that in the AO PR.

Reply:
On that note, I need to pay better attention when reverting stuff. That could’ve caused serious issues.

Reply:
Go ahead and fix it here, for reasons discussed on discord.